units/src/amount/signed.rs

```diff
@@ -192,6 +192,11 @@ impl SignedAmount {
     /// let amount = SignedAmount::from_sat(100_000);
     /// assert_eq!(amount.to_float_in(Denomination::Bitcoin), 0.001)
     /// ```
+    ///
+    /// # Panics
+    ///
+    /// Panics if the string representation of the amount cannot be parsed as a float.
+    /// This is unlikely since the string should always be a valid numeric value.
```

Comment:
In f0de79b1cb5ad53eaf78986ee61f4b878bec4c4d

It's not just unlikely, it's impossible. I think these new docs are misleading.

Reply:
I have changed this to allow `missing_panics_doc`